.vscode/launch.json

```diff
@@ -151,7 +151,12 @@
       "type": "chrome",
       "request": "attach",
       "name": "Attach to Electron Frontend",
-      "port": 9222
+      "port": 9222,
+      "webRoot": "${workspaceRoot}",
```

Comment:
Please use `workspaceFolder` rather than `workspaceRoot` as it is [deprecated](https://code.visualstudio.com/docs/editor/variables-reference#_why-isnt-workspaceroot-documented)